tasks/offlinePackagingTasks.ts

```diff
@@ -167,6 +174,26 @@ gulp.task(
     }, 'installDependencies')
 );
 
+// Defines a special task to acquire all the platform specific Razor packages.
+// All packages need to be saved to the consumption AzDo artifacts feed, for non-platform
+// specific packages this only requires running the installDependencies tasks.  However for Razor packages
+// we need to acquire the nuget packages once for each platform to ensure they get saved to the feed.
+gulp.task(
+    'updateRazorVersion',
+    // Run the fetch of all packages, and then also installDependencies after
+    gulp.series(async () => {
```

Comment:
this looks generally the same as `updateRoslynVersion`, consider sharing a function that passes the necessary component